stdlib/public/Concurrency/Executor.swift

```diff
@@ -15,24 +15,44 @@ import Swift
 /// A service that can execute jobs.
 @available(SwiftStdlib 5.1, *)
 public protocol Executor: AnyObject, Sendable {
+  // This requirement is repeated here as a non-override so that we
+  // get a redundant witness-table entry for it.  This allows us to
+  // avoid drilling down to the base conformance just for the basic
+  // work-scheduling operation.
+  @available(*, deprecated, message: "Implement enqueueJob instead")
   func enqueue(_ job: UnownedJob)
```

Comment:
TODO: the implemetations using the "old method" won't get a warning when they override... It may be good to offer a tailored diagnostic when people implement the not-move-only method.